### ISSUE TITLE
Get misc SoC Data not working anymore

### ISSUE THREAD

**Issue by cp1983** [edent/BMW-i-Remote#47]
Get misc SoC Data is not working anymore. I guess this has something to do with the link used to get it.

Error i get in from the server is:
The remote server returned an error: (403) Forbidden.'

Does anyone know how to get the SoC data now.

Still get the charge percentage. However not SOCmax and SOC

**Comment by rs38** [comment]
I had that issue 1-2 weeks ago and that was a temp. issue from BMW, as you was not able to login with the browser too.

**Comment by cp1983** [comment]
So you are able to get the State of charge now? For instance 19.3 kWh

Christian

Den ons. 21. nov. 2018, 08:26 skrev Falco <notifications@github.com:

**Comment by rs38** [comment]
true, currently I get 403, but can login via browser and see the SOC in the json: https://www.bmw-connecteddrive.de/api/vehicle/navigation/v1/WBxyz

**Comment by rs38** [comment]
there are still some issues with BMW, even their current app behaves strange, does not update. quite well known, if you google for CD problems.

**Comment by rs38** [comment]
it seems that BMW changed and devides the scope of their access token (which makes sense).
You now need an additional token that is only used for the portal based webservice calls like the one with `socmax`. there are some solutions in the wild in either PHP or python.
check e.g. https://github.com/sergejmueller/battery.ebiene.de/blob/master/_inc/api.class.php
@cp1983
